app/controllers/cards_controller.rb

```diff
@@ -1,7 +1,70 @@
 class CardsController < ApplicationController
 
   def index
-    @cards = Card.all
+    load_resources
   end
 
+  def show
+    load_resource
+  end
+
+  def new
+    build_resource
+  end
+
+  def create
+    build_resource
+    save_resource or render 'new'
+  end
+
+  def edit
+    load_resource
+    build_resource
+  end
+
+  def update
+    load_resource
+    build_resource
+    save_resource or render 'edit' 
```

Comment:
Use `||` instead of `or`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.